images/nginx/rootfs/build.sh

```diff
@@ -488,12 +474,10 @@ fi
 WITH_MODULES="--add-module=$BUILD_PATH/ngx_devel_kit-$NDK_VERSION \
   --add-module=$BUILD_PATH/set-misc-nginx-module-$SETMISC_VERSION \
   --add-module=$BUILD_PATH/headers-more-nginx-module-$MORE_HEADERS_VERSION \
-  --add-module=$BUILD_PATH/nginx-goodies-nginx-sticky-module-ng-$STICKY_SESSIONS_VERSION \
   --add-module=$BUILD_PATH/nginx-http-auth-digest-$NGINX_DIGEST_AUTH \
   --add-module=$BUILD_PATH/ngx_http_substitutions_filter_module-$NGINX_SUBSTITUTIONS \
   --add-module=$BUILD_PATH/lua-nginx-module-$LUA_NGX_VERSION \
   --add-module=$BUILD_PATH/lua-upstream-nginx-module-$LUA_UPSTREAM_VERSION \
-  --add-module=$BUILD_PATH/nginx_cookie_flag_module-$COOKIE_FLAG_VERSION \
```

Comment:
@aledbf why was this module removed?

Reply:
Since dynamic mode only, this module was not used by the ingress controller

Reply:
Not by the controller itself but it is useful to have this module and use it under `nginx.ingress.kubernetes.io/configuration-snippet: ` to add `secure` and `httponly` flags to certain cookies from 3rd party apps

Reply:
> Any chance to see this back in the image?

No.

We already have the foundation to add custom plugins to the controller without the need to change the base nginx image. Please check https://github.com/kubernetes/ingress-nginx/pull/3807 and as an example https://github.com/kubernetes/ingress-nginx/pull/3967
Once this is available, you could build a plugin using https://github.com/cloudflare/lua-resty-cookie 

I am sorry if this is not the answer you are looking for, but we are trying to allow custom features without the need of a custom docker image or the overhead we have maintaining a base image with all the nginx modules.

Reply:
I am also exploring alternatives to the current base image using multi-stage builds, like https://github.com/aledbf/horus-proxy/blob/master/Dockerfile to produce dynamic modules using openresty as the base image.